src/main/java/me/partlysanestudios/partlysaneskies/system/OneConfigScreen.java

```diff
@@ -41,6 +41,13 @@ public void resetBrokenStrings() {
     )
     public static boolean ignored;
 
+    @Info(
+        type = InfoType.INFO,
+        text = "Your mod Version: " + PartlySaneSkies.VERSION + ". Make sure to keep your mod updated to avoid bugs and get the newest features!",
```

Comment:
Can you use "You're on: " instead of "Your mod version"

Reply:
And also it feels kinda annoying to have it always there

Reply:
Could remove that, yeah
We could just have a /pssversion command and have the version in the help, and remove it from the config

Reply:
After thinking about it for some time, a command for the version would be better because we could display more information